sparkmagic/sparkmagic/kernels/pyspark3kernel/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.0.1'
```

Comment:
I will remove the version.
I cloned this from pysparkkernel. I can remove versions in pysparkkernel and sparkkernel in a different PR if you want.